modules/imgcodecs/src/grfmt_jpeg2000_openjpeg.cpp

```diff
@@ -606,7 +600,8 @@ bool Jpeg2KOpjDecoder::readData( Mat& img )
     switch (image_->color_space)
     {
     case OPJ_CLRSPC_UNKNOWN:
```

Comment:
Add a `/* FALLTHRU */` comment for this case.

Reply:
👍